docs/guide/advocacy.md

```diff
@@ -0,0 +1,83 @@
+# Advocacy
+
+Much of e18e's work involves making changes in existing open source repositories.
+These repositories generally have their own workflows that their maintainers expect to adhere to.
+It's important when suggesting changes to repositories to respect those workflows and actively work with those maintainers.
+
+## Coordinating Issues
+
+Many repositories prefer discussing potential changes in an issue before they can fully review a pull request.
+Unless a repository maintainer has explicitly indicated a willingness to bypass issue practices, stick with their normal flow of filing an issue and fully filling out the most applicable issue template.
+[Explaining Changes](#explaining-changes) covers the common information you'll want to include.
+
+### Umbrella e18e Tracking
+
+When many changes are possible for a single repository, please search for an "umbrella" issue in [e18e/ecosystem-issues](https://github.com/e18e/ecosystem-issues), and open one if it doesn't already exist.
+That will help with tracking progress and raising visibility for other e18e participants.
+
+We also use these issues to keep track of maintainer preferences around the initiative.
+If a maintainer requests a preference, especially one to not be sent e18e pull requests, please note it there.
+
+## Explaining Changes
+
+Most maintainers are not active participants in e18e.
+Many will have not heard of the initiative in the first place.
+Suggested changes need to include the contextual information that can clearly and quickly explain to a repository maintainer why the change is for the better.
+
+In general, it's best to demonstrate _tangible benefits_ that are _specific to the project_.
+That can change depending on what area(s) of improvement your change falls under.
+
+### Bundle Size
+
+When a project creates a built output such as bundled JS files, and a cleanup is meant to improve the size of those bundles, show the actual file size difference.
+
+This is important because:
+
+* Many changes might change the devDependency tree size, but not impact the bundle
+* Swapping out direct usage of a dependency that is also deeply required might result in a net bigger bundle
+
+In the latter case, you might consider prioritizing working on the upstream consumers of that dependency first.
+
+### Dependency tree
+
+If a dependency cleanup is meant to reduce the size of a project's dependency tree, show the actual change to the tree.
+
+Linking to just the npmgraph.js.org visualization of a package can be informative in isolation but doesn't directly indicate end-user benefits.
```

Comment:
I'm not quite sure what this sentence is suggesting and it sort of sounds to contradict the previous sentence. Is it that PR authors should also explain the degree to which reducing the dependency tree size reduces supply chain risk and saves disk space for users?